src/Parser/Monk/Windwalker/Modules/Spells/TouchOfKarma.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import Analyzer from 'Parser/Core/Analyzer';
+import HealingDone from 'Parser/Core/Modules/HealingDone';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Wrapper from 'common/Wrapper';
+
+class TouchOfKarma extends Analyzer {
+	static dependencies = {
+		combatants: Combatants,
+    abilityTracker: AbilityTracker,
+    healingDone: HealingDone,
+  };
+  totalPossibleAbsorb = 0;
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (SPELLS.TOUCH_OF_KARMA_CAST.id !== spellId){
+      return;
+    }
+    this.totalPossibleAbsorb += event.maxHitPoints * 0.5;
+  }
+
+  suggestions(when) {
+    const absorbUsed = this.healingDone.byAbility(SPELLS.TOUCH_OF_KARMA_CAST.id).effective / this.totalPossibleAbsorb;
+    const recommendedAbsorbUsed = 0.8;
+    when(absorbUsed).isLessThan(recommendedAbsorbUsed).addSuggestion((suggest, actual, recommended) => {
```

Comment:
I'd recommend using the new format for suggestions using the suggestionThresholds when writing new modules.